gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ReporterServiceImpl.java

```diff
@@ -201,19 +204,18 @@ public Single<Reporter> update(Reference reference, String reporterId, UpdateRep
                     if (!oldReporter.isSystem() || isUpgrader) {
                         reporterToUpdate.setConfiguration(updateReporter.getConfiguration());
                     }
+                    if (oldReporter.getReference().type() == ReferenceType.ORGANIZATION && !oldReporter.isSystem()) {
+                        // only non-system org reporters can be inherited
+                        reporterToUpdate.setInherited(updateReporter.isInherited());
+                    }
```

Comment:
**Question**: do we have to reject the request with 400 Bad Request if inherite is set to true for "system reporter" or domain reporter ? IMO better to notify the user of the error instead of silently accept ther request and ignore the value.